jscript.js

```diff
@@ -323,7 +323,11 @@ const messageval = document.getElementById('message');
 const emailError = document.querySelector('.erroremail');
 const nameerror = document.querySelector('.errorname');
 const messageerror = document.querySelector('.errormessage');
-
+// parsing data from local storage
+const storage = JSON.parse(localStorage.getItem('myLocalData'));
+nameval.value = storage.Name;
+email.value = storage.Email;
+messageval.value = storage.Message;
```

Comment:
- [ ]  Please make sure to check whether the local storage is empty or not before setting the input values as it currently throws an error in the console when accessing a null value.
  ![image](https://user-images.githubusercontent.com/104968236/233117268-e7f0d170-bcaa-439a-b975-dffe1dab2638.png)
   - You should add an `if` statement to check  if the local storage is empty before setting the values 😉